src/components/organisms/LocalSearch.tsx

```diff
@@ -51,6 +51,15 @@ interface LocalSearchProps {
 
 export type SearchResultNavigationDirection = 'next' | 'previous'
 
+export function scrollEditorToLine(
```

Comment:
Extract method from here. This method is now used for other purposes than local searching.

Reply:
Done